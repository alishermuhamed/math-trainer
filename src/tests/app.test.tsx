import React from 'react';
import App from '../app';
import renderer from 'react-test-renderer';

test('App component renders normally', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
