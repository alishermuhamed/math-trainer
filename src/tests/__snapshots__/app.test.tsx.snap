// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders normally 1`] = `
<div>
  Hello World
</div>
`;